pkg/config/constants.go

```diff
@@ -59,6 +59,10 @@ const (
 
 	// ControllerServiceAccountNameEnvVar stores the name of the serviceaccount used in the controller.
 	ControllerServiceAccountNameEnvVar = "CONTROLLER_SERVICE_ACCOUNT_NAME"
+
+	// WorkspaceStopReasonAnnotation marks the reason why the workspace was stopped; when a workspace is restarted
+	// this annotation will be cleared
+	WorkspaceStopReasonAnnotation = "io.devfile.devworkspace.stopped-by"
```

Comment:
I don't have a strong preference here but something like `controller.devfile.io/stopped-by` would be consistent with other labels/annotations we have. 
That's magic but maybe @amisevsk you have a better vision of what is the controller and what is devworkspace )

Reply:
I knew this was going to be discussed :smile: I just stole the `io.devfile.devworkspace.stopped-by` from the issue until another suggestion came

Reply:
I support the `controller.devfile.io/stopped-by` annotation -- lets finalize the usual naming discussion as quickly as possible :wink: 

For reference, the annotations we're currently using:

```
"controller.devfile.io/workspace_id"
"controller.devfile.io/endpoint_name"
"controller.devfile.io/workspace_name"
"controller.devfile.io/creator"
"controller.devfile.io/restricted-access"
"controller.devfile.io/discoverable-service"
```
so that would be in line with their format.

Reply:
Makes sense to me, I'll update both PRs